nav2_bt_navigator/behavior_trees/navigate_to_pose_w_replanning_goal_patience_and_recovery.xml

```diff
@@ -0,0 +1,49 @@
+
+<!--
+  This Behavior Tree replans the global path periodically at 1 Hz and it also has
+  recovery actions specific to planning / control as well as general system issues. 
+  This will be continuous if a kinematically valid planner is selected.
+  Additionally it has a feature to wait near the goal to have a new valid plan 
+  if the old plan becomes invalid before trying to drive significantly 
+  around by a length factor.
+-->
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <RecoveryNode number_of_retries="6" name="NavigateRecovery">
+      <PipelineSequence name="NavigateWithReplanning">
+        <RateController hz="1.0">
+          <RecoveryNode number_of_retries="1" name="ComputePathToPose">
+            <ComputePathToPose goal="{goal}" path="{path}" planner_id="GridBased"/>
+            <ClearEntireCostmap name="ClearGlobalCostmap-Context" service_name="global_costmap/clear_entirely_global_costmap"/>
+          </RecoveryNode>
+        </RateController>
+        <ReactiveSequence name="MonitorAndFollowPath">
+          <IsPathLongerOnApproach path="{path}" prox_leng="0.0" length_factor="1.0">
```

Comment:
I just left it for the user to decide the value. The reason was because initially we were planning to update it in the default BT right ? Anyways I can update it.